src/javaExercises/magicSquare/MagicSquare.java

```diff
@@ -0,0 +1,60 @@
+package javaExercises.magicSquare;
+
+public class MagicSquare {
+    static int N = 3;
+
+    public static boolean checkIfMagicSquare(int[][] square) {
```

Comment:
You need to add a few more checks:
   * You need to add a few more checks
   *  Numbers range {\displaystyle 1,2,...,n^{2}}{\displaystyle 1,2,...,n^{2}}
   * Each cell contains a different integer

Reply:
Done